src/test/java/itest/TargetRecordingOptionsIT.java

```diff
@@ -0,0 +1,266 @@
+                                                        "description",
+                                                        "Maximum size of recording",
+                                                        "defaultValue",
+                                                        "0B[B]"),
+                                                Map.of(
+                                                        "name",
+                                                        "Max Age",
+                                                        "description",
+                                                        "Maximum age of the events in the recording",
+                                                        "defaultValue",
+                                                        "0s[s]"),
+                                                Map.of(
+                                                        "name",
+                                                        "To disk",
+                                                        "description",
+                                                        "Record to disk",
+                                                        "defaultValue",
+                                                        "false"),
+                                                Map.of(
+                                                        "name",
+                                                        "Dump on Exit",
+                                                        "description",
+                                                        "Dump recording data to disk on JVM exit",
+                                                        "defaultValue",
+                                                        "false")))));
+
+        MatcherAssert.assertThat(getResponse.get(), Matchers.equalTo(expectedGetResponse));
+    }
+
+    @Test
+    @Order(2)
+    public void testGetTargetRecordingOptionsReturnsDefaultOptions() throws Exception {
+        CompletableFuture<JsonObject> getResponse = new CompletableFuture<>();
+        webClient
+                .get(OPTIONS_REQ_URL)
+                .send(
+                        ar -> {
+                            if (assertRequestStatus(ar, getResponse)) {
+                                MatcherAssert.assertThat(
+                                        ar.result().statusCode(), Matchers.equalTo(200));
+                                MatcherAssert.assertThat(
+                                        ar.result().getHeader(HttpHeaders.CONTENT_TYPE.toString()),
+                                        Matchers.equalTo(HttpMimeType.JSON.mime()));
+                                getResponse.complete(ar.result().bodyAsJsonObject());
+                            }
+                        });
+
+        JsonObject expectedGetResponse =
+                new JsonObject(Map.of("maxAge", 0, "toDisk", false, "maxSize", 0));
```

Comment:
Could you put a `FIXME` here with a link to #263? That's still something we should probably investigate further, although I am fairly convinced that it's just a JMC bug with how those OptionDescriptors are created or how the toDisk setting is read from the JVM.